### DISCUSSION TITLE
Fullscreen Heart Attack ending cutscenes

### DISCUSSION THREAD

**Discussion by Tenome** [emoose/OutRun2006Tweaks#239] [Suggestions]
### Describe the feature you'd like to see added

While using Cheat Engine to make a retexture for Clarissa, I noticed something interesting that you probably also already know about. Rather than the small window that normally plays the ending cutscene, with the credits on a black screen to the right, the ending was rendered in fullscreen. Can this be made into a Tweaks feature?

Normal:
![OR2006C2C_17259855134baac4eb-7e6d-4858-91ef-dacb485dca5a](https://github.com/user-attachments/assets/39a39432-b9d4-490c-8529-4a910a29818d)

Setting `observer flag` to 4.
![OR2006C2C_1725985109c4550a76-4ff2-45ab-b92a-cbc5dfe72134](https://github.com/user-attachments/assets/767d2c20-fde0-41a3-a583-33d5d96b3495)

**Comment by emoose** [comment]
Not really sure how game sets it to the smaller size yet, think it's something to do with D3D9 viewport being changed, but the only places I've seen that didn't seem to have any effect when I disabled it.. will look into it more soon, guess it's probably some flag thats set when finishing normally, but observer_flag itself wouldn't set.

Think these cutscenes might have some FOV issues too, noticed one did show Clarissa on the side in t-pose anyway (I think either in tulip garden or floral village cutscene)
IIRC there's some code in the game that tries to adjust camera FOV based on screen res, maybe that ends up causing issues with these cutscenes if you're not at 4:3.

**Comment by Tenome** [comment]
Yeah there are definitely things off-screen you're not meant to see. In the Goal A ending on Easter he'll pop in from the side (might've been Goal D Imperial Avenue).
